src/screens/User/User.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { View, Image, TouchableOpacity, Linking, FlatList } from 'react-native'
+import { Heading, HeadingType, PlainText } from 'src/components'
+import { Podcast, User } from 'src/common/types'
+import PodcastItem from './components'
+import { MockedUser, MockedPodcasts } from './mockedData'
+import Check from 'src/assets/images/checkMark.svg'
+import AtMark from 'src/assets/images/atMark.svg'
+import LogOut from 'src/assets/images/iconmonstr-log-out-16.svg'
+import styles from './styles'
+
+type Props = {
+  podcasts: Podcast[]
+  user: User
+}
+
+const userPageInfo: Props = {
+  podcasts: MockedPodcasts,
+  user: MockedUser
+}
+
+const UserPage: React.FC<Props> = () => {
+  const handleLogOut = () => {
+    return 0
```

Comment:
What does it mean `0`? Put it to `constants`.

Reply:
thats for now, I will do logic here in the next task. Thats jush 0. Nothing special